inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/RecommendationEditorExtension.java

```diff
@@ -168,14 +177,21 @@ private void actionRejectRecommendation(AnnotationActionHandler aActionHandler,
         Recommender recommender = recommendationService.getRecommender(aVID.getId());
         AnnotationLayer layer = annotationService.getLayer(aVID.getLayerId());
         AnnotationFeature feature = annotationService.getFeature(recommender.getFeature(), layer);
-        
-        AnnotationObject prediction = model.getPredictionByVID(aVID);
+
+        SourceDocument document = aState.getDocument();
+        Optional<AnnotationObject> oPrediction = model.getPredictionByVID(document, aVID);
+        if (!oPrediction.isPresent()) {
+            log.error("Could not find annotation in [{}] with id [{}]", document, aVID);
```

Comment:
Such a problem should also be logged to the user. We are usually in an interactive cycle when hitting this method.